lib/ContextualPopup/ContextualPopup.js

```diff
@@ -533,7 +549,7 @@ module.exports = kind(
 	* @private
 	*/
 	showHideScrim: function (inShow) {
-		if (this.floating && (this.scrim || (this.modal && this.scrimWhenModal))) {
+		if (this.floating && (this.scrim || this.modal || this.scrimWhenModal)) {
```

Comment:
Would you like to explain why we need this fix?
It looks like scrimWhenModal is meaning it should show scrim only when this.modal is true.